google-cloud-datastore/src/test/resources/index.yaml

```diff
@@ -0,0 +1,32 @@
+indexes:
+  - kind: TaskConcepts
+    properties:
+      - name: done
+      - name: priority
+        direction: desc
+  - kind: TaskConcepts
+    properties:
+      - name: tag
+      - name: tag
```

Comment:
IIUC, `tag` column should be listed only once.

Reply:
this file is required for [composite index configurations](https://cloud.google.com/datastore/docs/tools/indexconfig), In that case should `tag` (only one column) even be mentioned here?

Reply:
When I run `testInQuerySorted` without the index, I get the following error:
```
com.google.cloud.datastore.DatastoreException: no matching index found. recommended index is:
- kind: TaskConcepts
  properties:
  - name: tag
  - name: tag
 ```

I **assume** this is because of the orderBy on the query, but not sure. Listing `tag` once gives an additional error (that it is a single property index), so I think this is the correct solution given by the error message.